executor/cook/executor.py

```diff
@@ -444,7 +333,8 @@ def launch_progress_tracker(progress_location, location_tag):
     finally:
         # ensure completed_signal is set so driver can stop
         completed_signal.set()
-        kill_process_group(group_id, signal.SIGKILL)
+        if launched_process is not None and cs.is_process_running(launched_process):
```

Comment:
Could `launched_process is not None` be replaced with just `launched_process`?

Reply:
Yes, will do